### DISCUSSION TITLE
Nerdctl support

### DISCUSSION THREAD

**Discussion by rverma-dev** [JetBrains/Qodana#95] [Ideas]
Provide option to run qodana with nerdctl https://github.com/containerd/nerdctl. Usually nerdctl work as drop in replacement for docker but uses containerd.

**Comment by tiulpin** [comment]
Hi, thanks for writing the idea here, I created an issue in our tracker – https://youtrack.jetbrains.com/issue/QD-3341/Support-nerdctl. Please refer to it for the future updates
